vmdb/app/helpers/application_helper.rb

```diff
@@ -2680,14 +2680,15 @@ def placeholder_if_present(password)
 
   def render_listnav_filename
     if @lastaction == "show_list" && !session[:menu_click] &&
-       %w(ems_cloud ems_cluster ems_infra host miq_template offline orchestration_stack repository
+       %w(container_node container_service ems_container container_group ems_cloud ems_cluster
```

Comment:
Yes, @erinboyd, I'm concerned with the similar changes in different places.  Nothing you can really do about it here other than to note it and work with others to come up with future solutions to make this easier going forward.